worker/exec/konnector_test.go

```diff
@@ -361,5 +367,45 @@ echo "{\"type\": \"toto\", \"message\": \"COZY_URL=${COZY_URL}\"}"
 		assert.Contains(t, dir.CozyMetadata.CreatedOn, inst.Domain)
 		assert.Len(t, dir.CozyMetadata.UpdatedByApps, 1)
 		assert.Equal(t, dir.CozyMetadata.SourceAccount, acc.ID())
+
+		// Folder is created from Konnector name and Account name
+		wg.Add(1)
+		acc.DefaultFolderPath = ""
+		acc.Name = "account-1"
+		require.NoError(t, couchdb.GetDoc(inst, consts.Accounts, acc.ID(), acc))
```

Comment:
Why do you set some fields and then load the document from CouchDB? I would have expected to first load the doc from CouchDB and then set some fields, or set some fields and then save the doc.

Reply:
Hum that's a mistake. It should be a call to `UpdateDoc` here.